qiskit/compiler/transpiler.py

```diff
@@ -40,6 +50,17 @@
 
 _CircuitT = TypeVar("_CircuitT", bound=Union[QuantumCircuit, List[QuantumCircuit]])
 
+CUSTOM_MAPPING = {
+    "if_else": IfElseOp,
```

Comment:
The control flow instructions aren't handled by `Target.from_configuration()` already? That is arguably a bug in `from_configuration()`.

Reply:
True, this is actually not necessary with the current implementation.